aptos-node/src/utils.rs

```diff
@@ -55,16 +55,19 @@ pub fn set_aptos_vm_configurations(node_config: &NodeConfig) {
     } else {
         node_config.execution.concurrency_level
     };
-    AptosVM::set_concurrency_level_once(effective_concurrency_level as usize);
-    AptosVM::set_discard_failed_blocks(node_config.execution.discard_failed_blocks);
-    AptosVM::set_num_proof_reading_threads_once(
+    AptosVMStaticConfig::set_concurrency_level_once(effective_concurrency_level as usize);
+    AptosVMStaticConfig::set_committer_backup_enabled_once(
+        node_config.execution.enable_committer_backup,
+    );
+    AptosVMStaticConfig::set_discard_failed_blocks(node_config.execution.discard_failed_blocks);
+    AptosVMStaticConfig::set_num_proof_reading_threads_once(
         node_config.execution.num_proof_reading_threads as usize,
     );
```

Comment:
can we just have AptosVMStaticConfig::set_from_execution_configs_once(node_config.execution), which will set all of them?